### PR TITLE
components

### PR DESCRIPTION
**Pull request by cbflanagan3**
Added the components for product lists and pages and added some new products to the db :)

I ended up adjusting the App component because I was running into issues accessing the state. I rewrote some of the code in a way I could work with more easily, but I left the original code commented out. Happy to go with the original method and I'm sure it will be easy enough to switch back, but I may need a quick walkthrough!

### CODE REVIEW

**Comment by leafoflegend** [comment]
The commit is enormous, because of the `dist` folder. That needs to be `.gitignored`.

File: package.json [hunk 1]
```diff
@@ -25,16 +25,20 @@
...
   },
   "dependencies": {
     "axios": "^0.19.2",
     "bcrpyt": "^2.0.0",
     "body-parser": "^1.19.0",
+    "bootstrap": "^4.5.0",
     "cookie-parser": "^1.4.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jquery": "^3.5.1",
```
Comment (leafoflegend) [inline package.json:40]: ?

**Comment by jlam609** [comment]
Looks good, Great Job! mapState and mapDispatch are just a shortcut I use to not have to type the whole mapStateToProps and mapDispatchToProps, They are essentially the same thing though and can be renamed to w.e works for you. I can use class methods if it makes it more clear in the future. I added jquery to experiment with in the future but it can be disregarded for now. We can also use nodemon if we're not comfortable with nodedev. I will be pushing up a big file after work. I created authRouter and cartRouter as well as the cart prototype. I added a quantity key to the Products database and to the Cart database.

File: package.json (hunk 1 above)
Comment (jlam609) [reply package.json:40]: i was considering using jquery in the future for some experimental stuff but that wont be until much further down the line

**Pull request merged**